test/pom.xml

```diff
@@ -235,13 +269,18 @@
 		In newer versions of Eclipse (at least version 20181214-0600 onwards) this seems to have been resolved.
 		-->
 		<profile>
-			<id>m2e</id>
+			<id>spring-boot</id>
```

Comment:
Can the large comment block already be removed, line 259 - 270? Or in a later PR. 
It seems to be quite outdated.